packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -718,18 +718,7 @@ class _PopupMenuRouteLayout extends SingleChildLayoutDelegate {
     // size: The size of the overlay.
```

Comment:
This block of comments should be moved down, below `// Find the ideal horizontal position`, since they aren't needed to explain the y computation.